spyder/plugins/application/plugin.py

```diff
@@ -104,6 +134,21 @@ def on_statusbar_available(self):
         inapp_appeal_status = self.get_container().inapp_appeal_status
         statusbar.add_status_widget(inapp_appeal_status)
 
+    @on_plugin_available(plugin=Plugins.Toolbar)
+    def on_toolbar_available(self):
+        container = self.get_container()
+        toolbar = self.get_plugin(Plugins.Toolbar)
+        for action in [
+            container.new_action,
+            container.open_action,
+            container.save_action,
+            container.save_all_action
+        ]:
+            toolbar.add_item_to_application_toolbar(
+                action,
+                toolbar_id=ApplicationToolbars.File
```

Comment:
```suggestion
                toolbar_id=ApplicationToolbars.File,
                before=EditorWidgetActions.NewCell
```

As suggested by Carlos below.